### DISCUSSION TITLE
Filter by language

### DISCUSSION THREAD

**Discussion by danielh1267** [seerr-team/seerr#349] [Q&A] [answered]
Is it possible to add a filter so that only movies/shows of a desired language are shown? I tried using the discover region and discover language, but these settings do not filter based upon my setting (ie English).

**Comment by ajitesh-kuppa-sivakumar** [comment]
I am looking for this too! We need a option to download a media in one of the many language.

**Reply by fallenbagel** [reply]
The media acquisition of a specific language is and should be handled by *arr. You will have to configure it there.

**Reply by brianjmurrell** [reply]
Is this really what the OP was asking about?  I read his question being about not wanting jellyseerr to show him movies/series that are not natively in English.  That's what I am looking for, but have started a [new discussion](https://github.com/fallenbagel/jellyseerr/discussions/1297) about it in case it was not OP's original intention.

**Reply by fallenbagel** [reply]
> Is this really what the OP was asking about?  I read his question being about not wanting jellyseerr to show him movies/series that are not natively in English.  That's what I am looking for, but have started a [new discussion](https://github.com/fallenbagel/jellyseerr/discussions/1297) about it in case it was not OP's original intention.

My reply was for the comment made above. Not OPs.

**Comment by fallenbagel** [comment, marked as answer]
https://github.com/fallenbagel/jellyseerr/discussions/1297#discussioncomment-11919457
